ply-core/src/Ply/Core/Types.hs

```diff
@@ -5,19 +5,25 @@ module Ply.Core.Types (
   ScriptVersion (..),
   ScriptRole (..),
   ScriptReaderException (..),
-  TypedScriptEnvelope' (..),
   TypedScriptEnvelope (..),
   Typename,
 ) where
 
+import qualified Cardano.Binary as CBOR
```

Comment:
nit: Could you group up these imports here?

- [ ] Move `Codec.Serialise` import alongside the `Data.Aeson.Types` import group.
- [ ] Move the `Cardano.Binary` import below alongside the other `Cardano.Binary` import. 
- [ ] Move all `Ply.*` imports below alongside the `Ply.Core.Typename` import.
- [ ] Run `make format` afterwards.

`Ply.Core.Serialize.Script` could also use some import grouping. The import grouping rules I follow here are pretty vague tbh, but it's more or less:

- Haskell Core libraries as the first import group. (base, text, bytestring, containers)
- Other non-Cardano dependencies as the next import group. (aeson, serialise)
- Cardano/Plutus specific dependencies as the next import group. (cardano-*, plutus, plutarch)
- Local package modules as the final import group (ply stuff in this case)

(I may have mistakenly broken the above rules in some places since I can't enforce it with a tool)